lib/configService.js

```diff
@@ -51,6 +51,8 @@ function processEnvironmentVariables() {
             'IOTA_AMQP_EXCHANGE',
             'IOTA_AMQP_QUEUE',
             'IOTA_AMQP_DURABLE',
+            'IOTA_AMQP_RETRIES',
```

Comment:
Just to check...

This variables has been already include in documentation in a past PR?

Reply:
just added